tests/test_runs/test_run_functions.py

```diff
@@ -1618,7 +1618,9 @@ def test__run_task_get_arffcontent_2(self, parallel_mock):
             0.9655172413793104,
         ]
         scores = [v for k, v in res[2]["predictive_accuracy"][0].items()]
-        self.assertSequenceEqual(scores, expected_scores, seq_type=list)
+        np.testing.assert_array_almost_equal(
+            scores, expected_scores, decimal=2 if os.name == "nt" else 7
```

Comment:
I wasn't aware of this issue. This did not fail for me locally. Did you observe it in Appveyor?

Reply:
Yes

Reply:
Maybe that's due to the old python version of appveyor?

Reply:
No clue, could be any number of version differences. But if it's accurate to two decimals it's should be fine nonetheless.